stl/inc/xutility

```diff
@@ -3932,7 +4020,7 @@ namespace ranges {
         // clang-format on
     };
 
-    inline constexpr _Copy_fn copy{_Not_quite_object::_Construct_tag{}};
+    inline constexpr _Copy_fn copy{_Not_quite_object::_Construct_tag {}};

```

Comment:
GO HOME clang-format YOU ARE DRUNK (No change requested - just despair)

Hopefully my "match the number of `{` and `}`" suggestion will fix this.